pkg/grafana/library-element-handler.go

```diff
@@ -100,6 +100,14 @@ func (h *LibraryElementHandler) GetUID(resource grizzly.Resource) (string, error
 	return resource.Name(), nil
 }
 
+func (h *LibraryElementHandler) GetSpecUID(resource grizzly.Resource) (string, error) {
+	spec := resource["spec"].(map[string]interface{})
+	if val, ok := spec["uid"]; ok {
+		return val.(string), nil
+	}
+	return "", fmt.Errorf("UID not specified")
```

Comment:
All of these can be refactored to a helper function
```suggestion
         return specAttribute("uid")
```